browser/components/MarkdownPreview.js

```diff
@@ -225,7 +225,7 @@ export default class MarkdownPreview extends React.Component {
       const attachmentsAbsolutePaths = attachmentManagement.getAbsolutePathsOfAttachmentsInContent(noteContent, this.props.storagePath)
 
       files.forEach((file) => {
-        file = file.replace('file://', '')
+        file = file.replace('file:///', '')
```

Comment:
This fix doesn't work on macOS. I think we have to check which os is used. 
```js
if (global.process.platform === 'win32') {
  file = file.replace('file:///', '')
} else {
  file = file.replace('file://', '')
}
```

Reply:
Thanks for your review. I fixed it.